apps/arweave/src/ar_http_iface_middleware.erl

```diff
@@ -2994,15 +2994,25 @@ post_tx_parse_id(check_header, {Req, Pid, Encoding}) ->
 			end
 	end;
 post_tx_parse_id(check_ignore_list, {TXID, Req, Pid, Encoding}) ->
-	case ar_mempool:is_known_tx(TXID) of
-		true ->
-			{error, tx_already_processed, TXID, Req};
-		false ->
-			Ref = make_ref(),
-			erlang:put(tx_id_ref, Ref),
-			ar_ignore_registry:add_ref(TXID, Ref, 5000),
-			post_tx_parse_id(read_body, {TXID, Req, Pid, Encoding})
-	end;
+    case ar_mempool:is_known_tx(TXID) of
```

Comment:
This breaks the mechanism where we process every tx only once and thus makes the network amplify spam.

We would need to do something like a two-tier ignore registry where a tx with data is accepted if it is not yet recorded in the second tier.